lib/views/home/home_view.dart

```diff
@@ -330,7 +329,7 @@ class _HomeViewState extends State<HomeView> {
                   child: Padding(
                     padding: const EdgeInsets.symmetric(horizontal: 10),
                     child: Text(
-                      error,
+                      error!,
```

Comment:
This will crash if the code goes in the `else if (snapshot.hasData) {` because  `error` is null, right?

Reply:
Oh I got this error. This error occurs at an instance when we are fetching the data from the server. This condition hits then only. I have a fix for this.